### ISSUE TITLE
Focus the mouse pointer on the zoom.

### ISSUE THREAD

**Issue by Gabrielxd195** [LMMS/lmms#5614]
### Summary of improvements
Focus the mouse pointer with the zoom.

#### Justification
Currently when you zoom with ctrl plus the mouse wheel, the zoom is directed to the beginning of the track, it would be more practical to zoom in on the part where the mouse pointer is located as it does ardor.

#### Sketch

Current zoom behavior.
![Zoom de lmms](https://user-images.githubusercontent.com/26127693/89477874-60d8b780-d75c-11ea-8b62-12e0694003da.gif)

Expected zoom behavior (using reference ardor).
![Zoom de ardour](https://user-images.githubusercontent.com/26127693/89477909-7d74ef80-d75c-11ea-8b2c-3afd443c109d.gif)

**Comment by Spekular** [comment]
Already implemented in master.

**Comment by Gabrielxd195** [comment]
> Already implemented in master.

Dude, one question: when will you have master releases?

**Comment by qnebra** [comment]
> > Already implemented in master.
> 
> Dude, one question: when will you have master releases?

"Master" is just main branch in lmms repository with latest code. It depends on user to compile his own release with latest code, or just to wait to official release by team.

**Comment by Gabrielxd195** [comment]
> "Master" is just main branch in lmms repository with latest code. It depends on user to compile his own release with latest code, or just to wait to official release by team.

I understand that master is not recommended for everyday use, but in my opinion it would be good for all users to be up to date with all the corrections and features that are implemented in master that stable does not yet have. because not all of us know how to compile from the source code. what do you think?

**Comment by Spekular** [comment]
Nightly builds (releases compiled from master once a day) will likely be available at some point, but it's impossible to say when @Gabrielxd195

Some of the work required has been done already, but not all of it. I don't know what's left to do nor do I feel qualified to estimate how difficult/time consuming it'll be.

**Comment by Gabrielxd195** [comment]
> Some of the work required has been done already, but not all of it. I don't know what's left to do nor do I feel qualified to estimate how difficult/time consuming it'll be.

It must be hard to deal with all the work. they certainly do their best to keep this great app.

For my part I think (as a common user of lmms), that they prioritize more the releases of the master version, because lmms 1.3 is light years away, and currently master is not so affordable for the common user, and stable does not have the improvements and corrections that has a master.

Once released master, all the effort invested would be more appreciated by users. I recommend that you launch monthly instead of daily so as not to be so forced and to have a balance between the launch time and the number of improvements.

**Comment by Spekular** [comment]
> I recommend that you launch monthly instead of daily so as not to be so forced and to have a balance between the launch time and the number of improvements.

Nightlies would be automatically created, so this isn't a concern. Regular stable releases would also still be made, to be clear.

**Comment by Gabrielxd195** [comment]
> Nightlies would be automatically created, so this isn't a concern. Regular stable releases would also still be made, to be clear.

That's great. Friend I do not want to seem intrusive, but this topic should be the main one in discussion, since many users do not know the master branch or all the improvements it has, stable does not include them. many don't even know github. I think having continuous or more frequent launches would attract more users.

I opened this thread about zooming because I didn't know they had already implemented it in master, and I found out because I always test the pr binaries. If it were possible for the guys to release the master branch in binaries more frequently, I think this would cheer up many users and attract others. Thank you for answering my questions. regards
